server/repository/src/migrations/views/mod.rs

```diff
@@ -384,6 +385,27 @@ pub(crate) fn rebuild_views(connection: &StorageConnection) -> anyhow::Result<()
       CREATE VIEW requisitions_in_period AS
           SELECT 'n/a' as id, program_id, period_id, store_id, order_type, type, count(*) as count FROM requisition WHERE order_type IS NOT NULL
             GROUP BY 1,2,3,4,5,6;   
+
+  CREATE VIEW vaccination_card AS
+    SELECT 
+      vcd.id || pe.id AS id,
```

Comment:
```suggestion
      vcd.id || "_" ||  pe.id AS id,
```
(For purely cosmetic reasons ;) )